bigtop-deploy/puppet/hieradata/bigtop/cluster.yaml

```diff
@@ -191,6 +191,7 @@ hadoop::common::tez_jars: "/usr/lib/tez"
 #kafka
 kafka::server::port: "9092"
 kafka::server::zookeeper_connection_string: "%{hiera('bigtop::hadoop_head_node')}:2181"
+kafka::server::bind_addr: ""
```

Comment:
@johnsca What are we supposed to do instead? Do we just leave the value out of cluster.yaml?